ibis/expr/api.py

```diff
@@ -543,6 +546,44 @@ def asc(expr: ir.Column | str) -> ir.SortExpr | ops.DeferredSortKey:
         return ops.SortKey(expr).to_expr()
 
 
+def and_(*predicates: ir.BooleanValue) -> ir.BooleanValue:
+    """Combine multiple predicates using `&`.
+
+    Parameters
+    ----------
+    predicates
+        Boolean value expressions
+
+    Returns
+    -------
+    BooleanValue
+        A new predicate that evaluates to True if all composing predicates are
+        True. If no predicates were provided, returns True.
+    """
+    if not predicates:
+        return literal(True)
+    return functools.reduce(operator.and_, predicates)
+
+
+def or_(*predicates: ir.BooleanValue) -> ir.BooleanValue:
+    """Combine multiple predicates using `|`.
+
+    Parameters
+    ----------
+    predicates
+        Boolean value expressions
+
+    Returns
+    -------
+    BooleanValue
+        A new predicate that evaluates to True if any composing predicates are
+        True. If no predicates were provided, returns False.
+    """
+    if not predicates:
+        return literal(False)
+    return functools.reduce(operator.or_, predicates)
```

Comment:
Would `functools.reduce(operator.or_, predicates, literal(False))` work? Similarly for `and_`

Reply:
If you pass an initial value to `reduce`, it's always used, not just in the case of no elements. This means that these would be equivalent:

```
ibis.or_(a, b, c)

ibis.literal(false) | a | b | c
```

I didn't want `literal(False)`/`literal(True)` hanging around in every generated expression.